docs/source/i18n.rst

```diff
@@ -0,0 +1,80 @@
+i18n: internationalization of screen messages
+=============================================
+
+Plone already provides user-interface translations using the `plone.app.locales`_
+packages.
+
+In plone.restapi we also use those translations where the end user needs to have
+those translated strings, this way the front-end work is easier, because you
+directly get from the server everything you need, instead of needing to query
+yet another endpoint to get the translations.
+
+To do so, plone.restapi relies on Plone's language-negotiation configuration and lets
+Plone to do the work of deciding the language in which the messages should be shown.
+
+For the content of a multilingual site built using `plone.app.multilingual`_ this is
+an easy task: Plone is configured to show in the language of the content-object, so
+there is no need to ask anything to the REST API.
+
+Nevertheless, when you want to query the Plone Site object of a multilingual site, or
+any other endpoint in a plain Plone site with multiple languages configured, you need
+to query the REST API which language do you want to have the messages on, otherwise
+you will get the messages on the default language configured in Plone.
+
+To achieve that, the REST API requires to use the `Accept-Language` HTTP header passing
```

Comment:
Double backticks are needed for literal code markup in ReStructuredText 😉

Reply:
Thanks, I tend to mix Markdown and RST